api/api.go

```diff
@@ -543,6 +544,72 @@ func (api *api) GetNodeConnectionInfo(ctx context.Context) (*lnclient.NodeConnec
 	return api.svc.GetLNClient().GetNodeConnectionInfo(ctx)
 }
 
+func (api *api) GetAutoSwaps(ctx context.Context) (*GetAutoSwapsResponse, error) {
```

Comment:
This is more about the auto swaps configuration? would it make sense to rename? (GetAutoSwaps felt like it was fetching swaps that had been done)